### PR TITLE
assignment2-4

### PR DESCRIPTION
**Pull request by Simenkathir**

### CODE REVIEW

File: Assignment4/Simenkathir2-220864/task06.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/Simenkathir2-220864/task06.py:12]: this won't work in Python. It's notebook magic

File: Assignment4/Simenkathir2-220864/task07.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+''',
+initNs = {"rdfs":RDFS, "rdf":RDF,"ns":ns}
+)
+
+for r in g.query(q2):
+    print(r.individual)
+
+"""TASK 7.3: List all individuals of "Person" and all their properties including their class with RDFLib and SPARQL"""
+for s, p, o in g.triples((None, RDF.type, (ns.Person) )):
```
Comment (dgarijo) [inline Assignment4/Simenkathir2-220864/task07.py:71]: You are missing the subclasses from Person

File: Assignment4/Simenkathir2-220864/task07.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+for r in g.query(q1):
+    print(r.subclasses)
+
+
+"""ASK 7.2: List all individuals of "Person" with RDFLib and SPARQL (remember the subClasses)"""
+
+
+for s, p, o in g.triples((None, RDF.type, ns.Person)):
+    print(s)
```
Comment (dgarijo) [inline Assignment4/Simenkathir2-220864/task07.py:51]: missing the subclasses of person

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Comment by Simenkathir** [comment]
I have made the changes.

File: Assignment4/Simenkathir2-220864/task06.py (hunk 1 above)
Comment (dgarijo) [reply Assignment4/Simenkathir2-220864/task06.py:12]: This is not resolved. You have done nothing to change this line :)

**Review by dgarijo — APPROVED** [review]

**Comment by dgarijo** [comment]
Assignment 4 is ok. @ocorcho please merge if the rest is ok

**Pull request merged**